### PR TITLE
Page up/down support

### PR DESCRIPTION
**Pull request by yashpatel5400**
Added support to navigate image using page up+down keys

### CODE REVIEW

File: menus.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func FilterMenu() {
 	go func() {
 		for s := range stream {
 			config.Update("filterStr", s)
-			RefreshDisplay()
+			Display()
```
Comment (bcicen) [inline menus.go]: this change looks to be made mistakenly

**Comment by bcicen** [comment]
Thanks @yashpatel5400; please review my comment above.

The rest is working well in my testing -- please squash your multiple commits once the `Display()` change is corrected and we should be good to merge!

**Comment by yashpatel5400** [comment]
Awesome, let me quickly fix that up and push them!

**Comment by bcicen** [comment]
Great! Can you squash your multiple commits to one?

**Comment by yashpatel5400** [comment]
Oh wait, actually, found a bug in the version, that causes the program to crash when the window is small. I'll fix that up, squash the commits, and then push again.

**Comment by yashpatel5400** [comment]
Alright, fixed that -- let me squash the commits and then we'll be good

**Pull request closed without merging**